content/docs/deployment/helm/modules/installation/observability/_index.md

```diff
@@ -185,11 +187,6 @@ Below are the steps to deploy a new Grafana instance into your Kubernetes cluste
 
     ```yaml
     # grafana-values.yaml 
-    image:
```

Comment:
We don't need any replacement for these lines in the yaml?

Reply:
Originally removed this section so the latest version is downloaded. However, I realized that will pull the 11.x version of the image. Updated to use the version of Grafana (10.4.3) we tested.